french/maps/conv/evelyn_convs.rpy

```diff
@@ -4,131 +4,131 @@
-    d surprise ""
+    d surprise "Tu vas bien, mon amour ?"
 
 # game/maps/conv/evelyn_convs.rpy:18
 translate french evelyn_outback_fields_443e059a:
 
     # e nulla "Yeah, yeah, just...{w=0.5} It's taking me a while to get used to the weather and the...{w=0.5} Vastness of it."
-    e nulla ""
+    e nulla "Ouais, ouais, juste...{w=0.5} Il me faut du temps pour m'habituer à la météo et à...{w=0.5} Son immensité."
 
 # game/maps/conv/evelyn_convs.rpy:20
 translate french evelyn_outback_fields_8db2f802:
 
     # e nulla "It's a completely alien environment for a poor mountain bunny like me...{w=0.5} Even while doing Orphan work for Ozpin, I never left the Valesian half of the continent."
-    e nulla ""
+    e nulla "C'est un environnement complètement étranger pour une pauvre lapine des montagnes comme moi...{w=0.5} Même en faisant du travail d'Orphelin pour Ozpin, je n'ai jamais quitté la moitié Valésienne du continent."
 
 # game/maps/conv/evelyn_convs.rpy:21
 translate french evelyn_outback_fields_d2e573ae:
 
     # d amused "Thank god for creative applications of Aura...{w=0.5} And Polendina technology."
-    d amused ""
+    d amused "Dieu merci pour les applications créatives de l'Aura...{w=0.5} et la technologie Polendina."
 
 # game/maps/conv/evelyn_convs.rpy:23
 translate french evelyn_outback_fields_2ce66643:
 
     # e nulla "Sure, rub your fancy suit in my face...{w=0.5} I swear I'll steal it the next time you get naked."
-    e nulla ""
+    e nulla "C'est sur, frotte-moi ta combinaison de luxe au visage...{w=0.5} Je jure de la voler la prochaine fois que tu seras nu."
 
 # game/maps/conv/evelyn_convs.rpy:35
 translate french evelyn_outback_cliffs_6754d6aa:
 
     # e nulla "Fufufufu~"
-    e nulla ""
+    e nulla "Fufufufu~"
 
 # game/maps/conv/evelyn_convs.rpy:36
 translate french evelyn_outback_cliffs_34c54db8:
 
     # d smile "And what's got you into such a good mood?"
-    d smile ""
+    d smile "Et qu'est-ce qui te mets dans un si bonne humeur ?"
 
 # game/maps/conv/evelyn_convs.rpy:37
 translate french evelyn_outback_cliffs_f5c61404:
 
     # e nulla "Let's just say that I have a theory too...{w=0.5} Well, a couple, but the first isn't fully mine."
-    e nulla ""
+    e nulla "Disons juste que j'ai une théorie aussi...{w=0.5} En fait, plusieurs, mais la première n'est pas vraiment à moi."
 
 # game/maps/conv/evelyn_convs.rpy:39
 translate french evelyn_outback_cliffs_8d9c6a2f:
 
     # e nulla "One is about the Grimm.{w=0.3} The other is about our master and apprentice duo."
-    e nulla ""
+    e nulla "Une est à propos des Grimms.{w=0.3} L'autre est à propos de notre duo maître et apprenti."
 
 # game/maps/conv/evelyn_convs.rpy:40
 translate french evelyn_outback_cliffs_71474ee2:
 
     # d surprise "Oh.{w=0.5} Are you talking about that paper you and professor Port...?"
-    d surprise ""
+    d surprise "Oh.{=0.5} Est-ce que tu parle de ce papier toi et le professeur Port... ?"
```

Comment:
pareil qu'avant, omission du "that"
Est-ce que tu parles de ce papier que toi et le professeur Port... ?